src/main/java/TextView.java

```diff
@@ -20,9 +20,13 @@
 	public void update(Observable o, Object arg)
 	{
 		Model model = (Model) o;
+		
+		System.out.println("");
+		System.out.println("LOWERCASE: BLACK | UPPERCASE: WHITE ");
+		System.out.println(model.getWhosTurn() + "'S Turn ");
+		System.out.println("Turn Number: " +model.getTurnCount());
+		System.out.println("");
```

Comment:
- [ ] make a `drawHeader()` method. Put the code for displaying the key, who's turn and turn number into a method. Use a `StringBuilder` instance to build the string, instead of printing it out incrementally.